src/content/docs/glossary/glossary.mdx

```diff
@@ -59,12 +59,11 @@ Whether you're considering New Relic One or you're already using our capabilitie
     id="agent"
     title="agent"
   >
-    At New Relic, an agent is a piece of monitoring software that provides [integrations](#integration) with various technologies (for example, web frameworks, host operating systems, or database types). The agents send that data to New Relic, usually on a specific cadence.
+A New Relic agent is an installable piece of software that integrates with multiple types of technologies (for example, web frameworks, host operating systems, or database types) and reports data to New Relic, usually on a specific cadence. We have APM agents, an infrastructure agent, a browser monitoring agent, and mobile monitoring agents. The integrations included with our agents are just some of [our many integrations](https://newrelic.com/instant-observability). 
```

Comment:
I like the way you chopped up/expanded this paragraph. The first sentence captures an aspect of agents I hadn't realized I was unclear on. I have some style suggestions for the sentences unchanged from the original:

- When you list the kinds of agents ("We have APM agents, an infrastructure"), I might consider making the number on all the nouns consistent (either saying "We have an APM agent, infrastructure agent, browser monitoring agent etc etc" OR "We have APM agents, infrastructures agents etc etc."

- The last sentence feels a little recursive on first read. Can we rework it? Maybe something like "Our agents [are comptaible with?] one set of integrations, but we have many other kinds of integrations used by our other products." I think I'd want the sentence to be more clear on whether integrations are 'things' that exist outside New Relic that our agents are compatible with, or if they're 'things' created by New Relic for our agents that users can opt in/out of at the configuration stage.